### PR TITLE
Add ansible-pylibssh dependency

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>

##### SUMMARY

Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/911
Depends-On: https://github.com/ansible-collections/ansible.netcommon/pull/278

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by rohitthakur2590** [comment]
recheck

**Comment by Qalthos** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by ashwini-mhatre** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

File: changelogs/fragments/update_author_bgp_global.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+trivial:
+  - Update author content.
```
Comment (GomathiselviS) [inline changelogs/fragments/update_author_bgp_global.yaml:3]: Can you mention the addition of dependency?

**Review by GomathiselviS — APPROVED** [review]

**Pull request merged**